pkg/ytconfig/proxy.go

```diff
@@ -130,6 +131,9 @@ func getHTTPProxyServerCarcass(spec *ytv1.HTTPProxiesSpec) (HTTPProxyServer, err
 	c.RPCPort = consts.HTTPProxyRPCPort
 	c.MonitoringPort = ptr.Deref(spec.InstanceSpec.MonitoringPort, consts.HTTPProxyMonitoringPort)
 	c.Port = int(ptr.Deref(spec.HttpPort, int32(consts.HTTPProxyHTTPPort)))
+	c.ChytHttpServer = &HTTPServer{
```

Comment:
Let's add them only if http proxy spec has "chyt" field.

Reply:
Is this such solution that you means? Anyway, we have default ports for http(s).